framework/helpers/Url.php

```diff
@@ -15,4 +15,31 @@
  */
 class Url extends BaseUrl
 {
+    /**
+     * @param        $params  Array of values to convert into http query
+     * @param string $keyfather Inheritance of keys
+     * @param string $response
+     * @param bool   $ltrim
+     *
+     * @return string
+     */
+    public static function custom_http_build_query($params, $keyfather = '', $response = '', $ltrim = true) {
```

Comment:
This could be named just `http_build_query` without the _custom_ prefix and differences against the built-in function could be explained in phpdoc.